api/provider/aws/job_runner.go

```diff
@@ -27,19 +28,19 @@ func NewJobRunner(
 	l provider.LoadBalancerProvider,
 	s provider.ServiceProvider,
 	t provider.TaskProvider,
-	store job.Store,
```

Comment:
@sesh-kebab 
> Would you want to update the job status to failed here?

I've updated the runner to return a `string` representing the job's `Result`. Now this particular package doesn't even have access to the job store, so that should make the separation of responsibility more clear.